base/applications/cmdutils/more/lang/ro-RO.rc

```diff
@@ -12,10 +12,10 @@ STRINGTABLE
 BEGIN
     IDS_USAGE "Afișarea datelor pe ecran, pagină cu pagină.\n\
 \n\
-MORE [/E [opțiuni]] < [dispozitiv:][cale]numele fișierului\n\
+MORE [/E [opțiuni]] < [disc:][cale]numele fișierului\n\
 command | MORE [/E [opțiuni]\n\
 \n\
-    [dispozitiv:][cale]numele fișierului  Un fișier al cărui conținut\n\
+    [disc:][cale]numele fișierului        Un fișier al cărui conținut\n\
```

Comment:
A storage device could also be a SSD or NVMe, not necessarily a HDD as the word `disc` implies. IMHO `dispozitiv` better fits in this context because you take into account other kinds of storage mediums.